VideoWeb/VideoWeb.AcceptanceTests/Helpers/SeleniumEnvironment.cs

```diff
@@ -79,19 +78,15 @@ private IWebDriver InitSauceLabsDriver(string filename, ScenarioInfo scenario)
                     caps.SetCapability("version", "74.0");
                     caps.SetCapability("autoAcceptAlerts", true);
 
-                    var chromeOptions = new Dictionary<string, List<string>>
-                    {
-                        ["args"] = new List<string>
-                        {
-                            "use-fake-ui-for-media-stream",
-                            "use-fake-device-for-media-stream"
-                        }
-                    };
+                    var options = new ChromeOptions();
+                    options.AddArgument("ignore -certificate-errors");
```

Comment:
I believe there's an extra space here "ignore -certificate-errors" it should be "ignore-certificate-errors"

Reply:
Done, thanks :)